src/Modules/Settings/Dnn.PersonaBar.Prompt/Prompt.Web/src/components/TextLine.jsx

```diff
@@ -3,7 +3,10 @@ import Parser from "html-react-parser";
 import DomKey from "../services/DomKey";
 
 const TextLine = ({txt, css}) => {
-    const textLines = txt.split("\n");
+    if(!txt){
+        return null;
```

Comment:
Can returning null potentially break in another place?

Reply:
nope, null means nothing to render.